tests/test_updater.py

```diff
@@ -532,7 +532,7 @@ async def test_webhook_basic(
         # that depends on this distinction works
         if ext_bot and not isinstance(updater.bot, ExtBot):
             updater.bot = ExtBot(updater.bot.token)
-        if not ext_bot and not type(updater.bot) is Bot:
+        if not ext_bot and not isinstance(updater.bot, Bot):
```

Comment:
mh, here want to ensure that we don't have an `ExtBot` instance though … So either we tell ruff to ignore that or we use `and not isinstance(updater.bot, ExtBot)`

or we just don't run ruff on the test files. IISC we also don't run `mypy` and `pylint` on those …

Reply:
Oke, I will remove ruff to check tests